### ISSUE TITLE
Add Region Tutorial

### ISSUE THREAD

**Issue by johnthagen** [mozman/ezdxf#420]
Could a short tutorial for `Region` (`add_region()`) be added? I'm having a difficult time understanding what can be represented by it. If it's possible to represent polygons using a `Region`, a short tutorial would be great.

Perhaps the `Region` class docs could also expanded a bit to explain the function/purpose of this type? https://ezdxf.readthedocs.io/en/stable/dxfentities/region.html The linked AutoCAD docs unfortunately, don't shed much light on this.

Thanks.

**Comment by mozman** [comment]
The ACIS data is just stored or preserved by ezdxf. ACIS data (SAT or SAB) is a proprietary data format from Spatial Inc., there is no free documentation or open source libraries available, and this is the reason why there is no example or tutorial how to generate and add REGION, 3DSOLID, BODY, SURFACE, ....

**Comment by mozman** [comment]
New FAQ about this: https://ezdxf.mozman.at/docs/faq.html#how-to-add-edit-acis-based-entities-like-3dsolid-region-or-surface

**Comment by johnthagen** [comment]
Thanks.

Just a note on this part of the docs:

> there exist no free available documentation

I found a link from Wikipedia: https://en.wikipedia.org/wiki/ACIS#External_links

That links to: http://paulbourke.net/dataformats/sat/

Which has some kind of documentation and an example. Not sure if that is worth mentioning in the docs. Perhaps just the fact I commented on this issue will help someone in the future find it.
